### ISSUE TITLE
3.0 - Order Screens Headings/Buttons Should Be Updated for Accessibility

### ISSUE THREAD

**Issue by robincornett** [easydigitaldownloads/easy-digital-downloads#7796]
### Expected behavior
For semantic and accessible HTML markup, headings should only increase by one level, and buttons must have discernible text.

### Actual behavior
On the edit and add orders screens, the headings in the postbox elements are `<h3>`, and `<h2>` is skipped. We've addressed this on other screens; just need to do the same here.

On the add order screen, the "delete x item" buttons do not have discernible text (example):

```html
<button class="button-link delete">
	<span class="dashicons dashicons-no"></span>
</button>
```

### Steps to reproduce the behavior
1) Add a new order
2) Add an item and inspect the red X button, no text
3) Inspect the postbox headings

### Information (if a specific version is affected):
PHP Version: all
EDD Version (or branch): release/3.0
WordPress Version: all

**Comment by LisaCee** [comment]
PR #8152
